octodns_cloudflare/__init__.py

```diff
@@ -163,11 +169,10 @@ def zones(self):
             page = 1
             zones = []
             while page:
-                resp = self._try_request(
-                    'GET',
-                    '/zones',
-                    params={'page': page, 'per_page': self.zones_per_page},
-                )
+                params = {'page': page, 'per_page': self.zones_per_page}
+                if self.account_id is not None:
+                    params.update({'account.id': self.account_id})
```

Comment:
Minor nit, but with a single value assignment is preferable to update. 

```suggestion
                    params['account.id'] = self.account_id
```